code/code/disc/disc_shaman.cc

```diff
@@ -724,7 +724,9 @@ TTool *TBeing::getHeldTotem() {
 void TBeing::doSacrifice(const char *arg) {
   for (; isspace(*arg); arg++);
 
-  if (getPosition() != POSITION_STANDING) {
+  positionTypeT position = getPosition();
+
+  if (position != POSITION_STANDING || position != POSITION_FLYING) {
```

Comment:
Did you test it? How?
We really, really should get into a habit of writing unit tests.

Reply:
You can do greater than and less than comparisons with these enums too. https://github.com/sneezymud/sneezymud/blob/master/code/code/misc/enum.h#L157-L171

`if (getPosition() < POSITION_STANDING)` for instance would allow you to address everything besides standing, flying and mounted. Just FYI in case it's useful.